llama-index-integrations/llms/llama-index-llms-fireworks/llama_index/llms/fireworks/utils.py

```diff
@@ -12,17 +12,26 @@
     "accounts/fireworks/models/llama-v2-70b-chat": 4096,
     "accounts/fireworks/models/llama-v2-34b-code-instruct": 16384,
     "accounts/fireworks/models/llamaguard-7b": 4096,
+    "accounts/fireworks/models/llama-v3-8b-instruct": 8192,
+    "accounts/fireworks/models/llama-v3-70b-instruct": 8192,
```

Comment:
Wait I'm confused, why is this list missing all the llama3.1 models?

I can see them in the source code here
https://github.com/run-llama/llama_index/blob/0a4596d293c924cc2ebd8f893fa816858e0d997c/llama-index-integrations/llms/llama-index-llms-fireworks/llama_index/llms/fireworks/utils.py#L17

Reply:
Oh, i see your PR has merge conflicts, that would be why

Reply:
Added them.